databricks_cli/pipelines/api.py

```diff
@@ -60,6 +60,27 @@ def delete(self, pipeline_id, headers=None):
     def get(self, pipeline_id, headers=None):
         return self.client.get(pipeline_id, headers)
 
+    def list(self, max_results=None, order_by=None, headers=None):
```

Comment:
Can we just get rid of all these default parameters if we're not using them yet? eventually these should come from user args, but better to add them then than to have them sit around unused now.

Reply:
They're useful for unit testing, fwiw. It's useful to ensure they're plumbed through as expected.

Reply:
Plumbed through from where to where? You're testing a variants of the API that are never actually used in the production code.

Reply:
Making sure the perform_query calls have the right params. You're right it doesn't change much, so I'll remove it for now.

Reply:
Done.